.github/workflows/odoo-ci-cd.yml

```
@@ -106,6 +134,129 @@
+            
+            # Wait for GraphSync to be ready
+            echo "Checking GraphSync service..."
+            for i in {1..30}; do
+              if docker exec graph_sync curl -s "$GRAPHSYNC_URL/healthcheck" >/dev/null 2>&1; then
+                echo "✅ GraphSync service is running"
+                break
+              fi
+              echo "Waiting for GraphSync... ($i/30)"
+              sleep 2
+            done
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)